ts/private/ts_project.bzl

```diff
@@ -141,6 +142,17 @@ See https://github.com/aspect-build/rules_ts/issues/361 for more details.
     tsconfig_inputs = copy_files_to_bin_actions(ctx, _validate_lib.tsconfig_inputs(ctx).to_list())
     inputs.extend(tsconfig_inputs)
 
+    assets_outs = []
+    for a in ctx.files.assets:
+        a_path = _lib.relative_to_package(a.short_path, ctx)
+        a_out = _lib.to_out_path(a_path, ctx.attr.out_dir, ctx.attr.root_dir)
+        if a_out != a:
```

Comment:
Would this work without the conditional? You could just use

```
            asset = ctx.actions.declare_file(a_out)
            copy_file_action(ctx, a, asset)
```
in either case and it would mimic the copy_to_bin behavior if the output path was the same

Reply:
Yeah that works and simplifies things. Logically I think it ends up doing the exact same thing but reduces the code here 👍